Zend/zend_API.c

```diff
@@ -1289,14 +1289,14 @@ ZEND_API int _object_and_properties_init(zval *arg, zend_class_entry *class_type
 			zend_error(E_EXCEPTION | E_ERROR, "Cannot instantiate abstract class %s", class_type->name->val);
 		}
 		ZVAL_NULL(arg);
-		Z_OBJ_P(arg) = NULL;
+		_Z_OBJ_P(arg) = NULL;
```

Comment:
I think, Z_OBJ_P() and all Z_...() macros should be the part of recommended zval API, they were used for ages. I don't see a reason to rename them into _Z...().